modules/kubernetes/ingress-nginx/templates/values.yaml.tpl

```diff
@@ -49,6 +49,10 @@ controller:
     %{~ if !allow_snippet_annotations ~}
     annotation-value-word-blocklist: load_module,lua_package,_by_lua,location,root,proxy_pass,serviceaccount,{,},',\
     %{~ endif ~}
+    deny-default-annotation-blocker: ${deny-default-annotation-blocker}
```

Comment:
Unsure why you add this config. Is it a part of the helm chart that is missing for some reason?